lib/xs_transport_lwt_unix_client.ml

```diff
@@ -45,7 +45,7 @@ let destroy = Lwt_unix.close
 let read = Lwt_unix.read
 
 let write fd bufs ofs len =
-  Lwt_unix.write_string fd bufs ofs len >>= fun n ->
+  Lwt_unix.write_string fd (Bytes.to_string bufs) ofs len >>= fun n ->
```

Comment:
This might do a copy, but I guess the performance of this call doesn't matter that much, so we don't have to  consider using `Bytes.unsafe_to_string`?

Reply:
The documentation does not recommend it: http://caml.inria.fr/pub/docs/manual-ocaml/libref/Bytes.html
It is probably safe to use here though. I'll change it

Reply:
Yeah, I guess in general we should only use these unsafe functions if not using them has a performance impact.